package.json

```diff
@@ -49,7 +50,6 @@
     "vue-loader": "17.3.1",
     "vue3-calendar-heatmap": "2.0.5",
     "webpack": "5.89.0",
-    "webpack-cli": "5.1.4",
```

Comment:
`npm uninstall webpack-cli && npm i webpack-cli` to keep it in `dependencies` please. We keep everything that is required for the webpack build in dependencies.

Reply:
Sorry, not even sure why it was moved by npm.